functionalTest/commandLine/imposters/imposters.ejs

```diff
@@ -2,6 +2,7 @@
     "imposters": [
         <% include accounts.json %>,
         <% include email.json %>,
-        <% include orders.json %>
+        <% include orders.json %>,
+        <% include users.json %>
```

Comment:
Test fails if I move users.json to a subdirectory, and users.json references users.xml locally within that subdirectory. Looking into it now. (I'll leave that in, it makes a better test that way anyhow and people have definitely complained about that type of scenario before).

Reply:
is `users.json` in a subdirectory and `users.xml` in the directory above that subdirectory?

Reply:
No, both in the same dir. However, I think this was the behavior before the change too -- I may be confused. I thought the stringify function was always relative to the current file, which feels right to me, but I think it was always relative to the root EJS file. I'm looking through old support troubleshooting emails to see if I can track down previous expectations.

This is old code. Sometimes I just forget how stuff works ;>

Reply:
OK, confirmed old behavior was always relative to the root EJS dir, and your change maintains that behavior. Thank you.

Now that I've got my head wrapped around it, I think this is the time to correct to the more intuitive behavior of being relative to the current file (like the EJS include directive). This means:
1. If filename is passed, maintain old behavior for backwards compatibility
2. If no filename, ensure that the path is interpreted relative to the current file path.

I'm playing with passing the context in a closure, creating a new stringify function on each invocation to see if that works for the 2nd point.

Reply:
After some research, I don't think it's possible to do what I want to do with EJS. Posted a StackOverflow here just to be sure: https://stackoverflow.com/questions/56658196/how-do-i-get-current-file-name-for-functions-passed-to-ejs-render

I'll give it a couple days to see if I get an answer, then merge as is if not. Thanks again.